packages/widgets/src/Pagination/PageSummary.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import { spacers } from '@dhis2/ui-constants'
+
+import { sectionStyles } from './Pagination.styles.js'
+
+const getItemRange = (page, pageSize, total) => {
+    const firstItem = total === 0 ? 0 : (page - 1) * pageSize + 1
+    const lastItem = Math.min(firstItem + pageSize - 1, total)
+
+    return { firstItem, lastItem }
+}
+
+const PageSummary = ({ dataTest, page, pageCount, pageSize, total }) => {
+    const { firstItem, lastItem } = getItemRange(page, pageSize, total)
+
+    return (
+        <div className="section" data-test={`${dataTest}-summary`}>
+            <span>
+                {i18n.t(
+                    'Page {{page}} of {{pageCount}}, item {{firstItem}}-{{lastItem}} of {{total}}',
```

Comment:
I think `item` should be `items` here. There is a chance there is only a single item, but more often there will be multiple I think.